pkg/converter/networkpolicy_converter_test.go

```diff
@@ -389,23 +395,23 @@ var _ = Describe("NetworkPolicy conversion tests", func() {
 
 		// Assert policy name.
 		By("returning a calico policy with expected name", func() {
-			Expect(pol.(api.Policy).Metadata.Name).To(Equal("knp.default.default.testPolicy"))
+			Expect(pol.(api.NetworkPolicy).Name).To(Equal("knp.default.default.testPolicy"))
```

Comment:
nit: i know this is not really a v2 change, but we could save a lot of casting for every case if we save the casted type when we get it. i.e. `policy := pol.(api.NetworkPolicy)` then we can just do `policy.Spec.Order` etc

Reply:
Yeah, that's true, I'm going to leave as is for now.